js/binance.js

```diff
@@ -139,6 +140,22 @@ module.exports = class binance extends Exchange {
                         'sub-account/assets',
                     ],
                 },
+                'fapiPublic': {
+                    'get': [
+                        'ping',
+                        'time',
+                        'exchangeInfo',
+                        'depth',
+                        'trades',
+                        'historicalTrades',
+                        'aggTrades',
+                        'klines',
+                        'premiumIndex',
+                        'ticker/24hr',
+                        'ticker/price',
+                        'ticker/bookTicker',
```

Comment:
```suggestion
                        'ticker/bookTicker',
                        'income',
```
Could you please add this API definition? It would really help me on my project! :pray:

Reply:
@ftuyama unfortunately, I don't have a push permission to this branch, but I think @StanleyDing has. We hope to merge this PR soon, however, we will rework the market loading sequence a bit and the default type option as in `okex3`, since `type` is a property of a market.

Reply:
@ftuyama, actually, I've added it, no worries )